include/caf/events/ble_common_event.h

```diff
@@ -22,64 +23,163 @@
+	 *  used for a confirmed operation.
+	 */
 	PEER_OPERATION_ERASE,
+
+	/** Trigger erase advertising for selected local identity. */
 	PEER_OPERATION_ERASE_ADV,
+
+	/** Cancel ongoing erase advertising. */
 	PEER_OPERATION_ERASE_ADV_CANCEL,
+
+	/** The bond erase is confirmed. That results in removing all of the Bluetooth bonds of
+	 *  related local identity.
+	 */
 	PEER_OPERATION_ERASED,
+
+	/** Cancel ongoing peer operation. If PEER_OPERATION_ERASE_ADV is cancelled,
+	 *  PEER_OPERATION_ERASE_ADV_CANCEL must be used instead.
+	 */
 	PEER_OPERATION_CANCEL,
 
+	/** Number of peer operations. */
 	PEER_OPERATION_COUNT
 };
 
-/** @brief BLE peer event. */
+/** @brief Bluetooth LE peer event.
+ *
+ * The Bluetooth LE peer event is submitted to inform about state change of a Bluetooth connected
+ * peer.
+ */
 struct ble_peer_event {
+	/** Event header. */
 	struct event_header header;
 
+	/** State of the Bluetooth LE peer. */
 	enum peer_state state;
+
+	/** ID used to identify Bluetooth connection - pointer to the @ref bt_conn structure. */
 	void *id;
 };
 
-/** @brief BLE peer operation event. */
+/** @brief Bluetooth LE peer operation event.
+ *
+ * The Bluetooth LE peer operation event informs about operation that is performed on Bluetooth
+ * peers related to a given Bluetooth local identity (and application local identity).
+ *
+ * For detailed information related to local identities, peer and bond management, see documentation
+ * of the Bluetooth LE bond module in nRF Desktop. The Bluetooth LE bond module is an example of
+ * application-specific module that implements Bluetooth peer and bond management.
+ */
 struct ble_peer_operation_event {
+	/** Event header. */
 	struct event_header header;
 
+	/** Bluetooth LE peer operation. */
 	enum peer_operation op;
+
+	/** Local identity related to the peer operation (used by the application). */
 	uint8_t bt_app_id;
+
+	/** Bluetooth local identity related to the peer operation (used by the Bluetooth stack). */
 	uint8_t bt_stack_id;
 };
 
-/** @brief BLE connection parameters event. */
+/** @brief Bluetooth LE connection parameters event.
+ *
+ * The Bluetooth LE connection parameters event is submitted to inform either about received
+ * Bluetooth connection parameter update request or fact that connection parameters were already
+ * updated.
+ *
+ * If the event is related to fact that the connection parameters were already updated, values
+ * of interval_min and interval_max are the same. They are equal to the connection interval that
+ * is in use.
+ */
 struct ble_peer_conn_params_event {
+	/** Event header. */
 	struct event_header header;
 
+	/** ID used to identify Bluetooth connection - pointer to the @ref bt_conn structure. */
 	void *id;
+
+	/** Minimum Connection Interval (N * 1.25 ms). */
 	uint16_t interval_min;
+
+	/** Maximum Connection Interval (N * 1.25 ms). */
 	uint16_t interval_max;
+
+	/** Connection Latency. */
```

Comment:
Does it have a unit?

Reply:
It is a number of connection intervals that peripheral can skip (I think it's known fact so no further explanation is needed).
It's also described in similar way in Zephyr.